isis/src/mro/apps/himos/himos.cpp

```diff
@@ -0,0 +1,317 @@
+#include "himos.h"
+
+#include "Camera.h"
+#include "Cube.h"
+#include "FileList.h"
+#include "IException.h"
+#include "Pvl.h"
+#include "IString.h"
+#include "Longitude.h"
+#include "OriginalLabel.h"
+#include "Process.h"
+#include "ProgramLauncher.h"
+#include "TProjection.h"
+
+#include <vector>
+
+using namespace std;
+
+namespace Isis {
+  //functions in the code
+  void CompareLabels(Pvl &match, Pvl &comp);
```

Comment:
just to avoid similar problems as what I ran into with lronac2isis (the duplicate symbols, not PR garbage), probably should start making these static.

Reply:
Like, move them out of the namespace Isis section or add the static declaration?

Reply:
just add static, I don't think it matters if it's in the Isis namespace if it's static, it should be scoped to the cpp either way.